docs/data-formats.md

```diff
@@ -308,21 +308,21 @@ This allows to add a custom attribute to a class, interface, delegate, struct or
     <type fullname="Assembly.A">
       <method signature="System.Void Method1(System.Type)">
         <parameter name="typeParameter">
-          <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers">
+          <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers" assembly="AssemblyName">
```

Comment:
```suggestion
          <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute" assembly="System.Runtime">
```

I know we accept the attribute names without the `Attribute` suffix, but for a sample code I would prefer to use the full correct type name (it makes it clear that this is a type name).

Specifically for `DynamicallyAccessedMembersAttribute` we know that this lives in `System.Runtime` assembly (the implementation is in S.P.CoreLib but System.Runtime has a typeforwarder to it and it's the public assembly which apps should refer to).